src/modules/tags/TagDetail.js

```diff
@@ -39,6 +46,29 @@ const TagDetail = (props) => {
     }
   };
 
+  const renderSpotFeatureItem = (spotFeature) => {
+    const spot = useSpots.getSpotById(spotFeature.spotId);
+    if (!isEmpty(spot)) {
+      return (
+        <ListItem
+          containerStyle={commonStyles.listItem}
+          onPress={() => props.openSpot(spot)}
+        >
+          <Avatar
+            source={useSpots.getSpotDataIconSource(deepFindFeatureTypeById(spot.properties,spotFeature.id))}
+            placeholderStyle={{backgroundColor: 'transparent'}}
+            size={20}
+          />
+          <ListItem.Content>
+            <ListItem.Title style={commonStyles.listItemTitle}>{spotFeature.label}</ListItem.Title>
```

Comment:
This is giving `Unknown Name` because no one really uses the label. The display should be the same as how it is displayed in the feature list in the notebook. In fact it should even use the same component to display a feature item.

Reply:
needs discussion.